From b402f9e01f84bd77a640dc1ce31c69399e2378fb Mon Sep 17 00:00:00 2001
From: Tanu Kaskinen <tanu.kaskinen@linux.intel.com>
Date: Sat, 12 Oct 2013 12:45:58 +0300
Subject: [PATCH 102/103] build-sys: Print CPPFLAGS in configure

I needed this for verifying that the conditional defining of
_FORTIFY_SOURCE works.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index e2465e4..72e7695 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1438,6 +1438,7 @@ echo "
     System Config Path:            ${PA_SYSTEM_CONFIG_PATH}
     Compiler:                      ${CC}
     CFLAGS:                        ${CFLAGS}
+    CPPFLAGS:                      ${CPPFLAGS}
     LIBS:                          ${LIBS}
 
     Enable X11:                    ${ENABLE_X11}
-- 
1.8.4

